Bottom: aa44e315a9ae82e74cb466c2ca69f0eb554ed42f
Top:    50bce11b6afb0dd1c53dee3a75561cfdef2b85c2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-27 22:09:10 +0100

Refresh of place-drawing-of-crosshair-att

---

diff --git a/src/crosshair.c b/src/crosshair.c
index 24f51ab..f733e11 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -85,7 +85,6 @@ static void XORDrawMoveOrCopyObject (void);
 static void XORDrawAttachedLine (LocationType, LocationType, LocationType,
 				 LocationType, BDimension);
 static void XORDrawAttachedArc (BDimension);
-static void DrawAttached (bool);
 
 /* ---------------------------------------------------------------------------
  * creates a tmp polygon with coordinates converted to screen system
@@ -566,7 +565,7 @@ XORDrawMoveOrCopyObject (void)
 /* ---------------------------------------------------------------------------
  * draws additional stuff that follows the crosshair
  */
-static void
+void
 DrawAttached (void)
 {
   BDimension s;
diff --git a/src/crosshair.h b/src/crosshair.h
index b07af6f..b4f2a71 100644
--- a/src/crosshair.h
+++ b/src/crosshair.h
@@ -46,7 +46,7 @@
 #define	STATE_SECOND	1
 #define	STATE_THIRD		2
 
-
+void DrawAttached (bool);
 void CrosshairOn (bool);
 void CrosshairOff (bool);
 void HideCrosshair (bool);
